Client/src/Pages/Auth/Login.jsx

```diff
@@ -384,7 +272,7 @@ const Login = () => {
 		password: DEMO !== undefined ? "Demouser1!" : "",
 	});
 	const [errors, setErrors] = useState({});
-	const [step, setStep] = useState(0);
+	const [step, setStep] = useState(1);
```

Comment:
Let's stick with 0 indexing as everything else is 0 indexed, we should be consistent.

Reply:
Fixed in: https://github.com/bluewave-labs/checkmate/commit/76bfb7717ac3eada7d94e11a4ce9291119b45081